OpenSim/Simulation/Model/Muscle.cpp

```diff
@@ -144,6 +157,7 @@ void Muscle::constructProperties()
     // By default the min and max controls on muscle are 0.0 and 1.0
     setMinControl(0.0);
     setMaxControl(1.0);
+    upd_GeometryPath().setDefaultColor(DefaultMuscleColor);
```

Comment:
It looks like this is here to force Muscle to have a default color of red regardless of its `default_color` property value. Is that right, @aymanhab?  Is that really necessary? Since muscles in the past did not have a default color and now the consumers of `GeometryPath` provide reasonable defaults, why force it here? If someone want to change the `default_color` then why prevent it? I don't think old models will accidentally visualize as not Red.